dali/operators/image/remap/cvcuda/matrix_adjust.cu

```diff
@@ -54,7 +54,7 @@ void adjustMatrices(nvcv::Tensor &matrices, cudaStream_t stream) {
 
   int num_blocks = div_ceil(bs, 256);
   int threads_per_block = std::min(bs, 256);
-  adjustMatricesKernel2<<<num_blocks, threads_per_block, 0, stream>>>(wrap, bs);
+  CUDA_CALL(adjustMatricesKernel2<<<num_blocks, threads_per_block, 0, stream>>>(wrap, bs));
```

Comment:
I don't think this would compile. A CUDA kernel invocation doesn't return a value.
Use `CUDA_CALL(cudaGetLastError()` after the kernel invocation.